recipes/cosmosis-standard-library/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "cosmosis-standard-library" %}
+{% set version = "2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/joezuntz/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fc815e5068a1e212c1af8af2106c42e14881f466b462c0a6c918a144e5fbb30c
+
+build:
+  number: 0
+  skip: True  # [win or (py < 36)]
+  detect_binary_files_with_prefix: True
+  missing_dso_whitelist:
+    - "*/liblimber.so"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - cosmosis
+    - cython
+    - mpich          # [mpi == "mpich"]
+    - openmpi        # [mpi == "openmpi"]
```

Comment:
These mpi selectors don't do anything here since they are only evaluated at run time. Please see the conda-forge docs for how to build MPI variants for your packages. https://conda-forge.org/docs/maintainer/knowledge_base.html#message-passing-interface-mpi